org.recoll.recoll.yml

```diff
@@ -0,0 +1,124 @@
+id: org.recoll.recoll
+runtime: org.kde.Platform
+runtime-version: 5.15-23.08
+sdk: org.kde.Sdk
+base: io.qt.qtwebkit.BaseApp
+base-version: 5.15-23.08
```

Comment:
we should create a 24.08 branch of this if it still builds.

Reply:
Do you need to use QtWebkit? This is unmaintained and not in upstream Qt tree. Also, see https://blogs.gnome.org/mcatanzaro/2022/11/04/stop-using-qtwebkit/

It might get stuck on an EOL runtime and so will your application.

Reply:
OK, I'll try using QtWebEngine instead.